osmpy/core.py

```diff
@@ -34,26 +35,29 @@ def get_area(s):
     proj = partial(
         pyproj.transform, pyproj.Proj(init="epsg:4326"), pyproj.Proj(init="epsg:3857")
     )
-
-    return transform(proj, s).area / 1e6  # km
+    area = transform(proj, s).area / 1e6  # km
+    if pd.isnull(area):
```

Comment:
This function is called internally. Maybe we can check the coordinates format at the `get` function to avoid error propagation.

Reply:
That would be awesome. Unfortunately, I'm not very familiar with `shapely`, so I didn't know a simple way to perform this check. As a result, I came up with a partial solution - raising an error in this function and letting the user know what caused it.

Reply:
ChatGPT response to `check if coordinates are in the correct order with shapely python`

```
To check if coordinates are in the correct order with Shapely in Python, you can use the is_ccw method of the LinearRing class. This method returns True if the coordinates are in a counterclockwise order, and False if they are in a clockwise order.

Here is an example of how to use is_ccw to check the order of coordinates:

```
from shapely.geometry import LinearRing

# Coordinates in a counterclockwise order
coords = [(0, 0), (0, 1), (1, 1), (1, 0)]
ring = LinearRing(coords)
print(ring.is_ccw)  # Output: True

# Coordinates in a clockwise order
coords = [(1, 0), (1, 1), (0, 1), (0, 0)]
ring = LinearRing(coords)
print(ring.is_ccw)  # Output: False
```

Note that is_ccw only works for LinearRing objects, which are closed sequences of coordinates that represent a polygon. If you have a sequence of unconnected coordinates, you can use the orient function from the shapely.geometry.polygon module to determine the orientation of the coordinates.

```
from shapely.geometry import orient

# Coordinates in a counterclockwise order
coords = [(0, 0), (0, 1), (1, 1), (1, 0)]
print(orient(coords))  # Output: 1

# Coordinates in a clockwise order
coords = [(1, 0), (1, 1), (0, 1), (0, 0)]
print(orient(coords))  # Output: -1
```

Reply:
Apparantely it just checks if the coordinates are in clockwise or counterclockwise order. Instead, we need to ensure that the polygon is inside the "box" `[(-180, -90), (-180, 90), (180, 90), (180, -90)]`

Reply:
Shapely has a built-in method for it:

```
from shapely.geometry import Point

# Create a point object with invalid latitude longitude coordinates
point = Point(90.1, -180.1)

# Check if the point has valid coordinates and fix any issues if necessary
is_valid, issues = point.validate()
if is_valid:
    print("The point has valid coordinates.")
else:
    print("The point does not have valid coordinates. Issues:", issues)
```

Reply:
If you can merge the branches, I can later add this kind of check.